ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-v1.11-periodics.yaml

```diff
@@ -16,18 +16,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-redhat-developer-gitops-operator-v1.10-gitops-ocp4.14-lp-interop-redhat-openshift-gitops-interop-aws
-  reporter_config:
```

Comment:
Could you please add this section (lines 20-30) back to this file instead of removing it?

When we make a change to a job name then run `make update` it does not account for this slack plugin and it will regenerate the prowjob file without it included. We then need to manually add this reporter_config section to the new prowgen create prowjob file.

this is due to prowgen using the ci-operator and the ci-operator does not officially support the slack plugin, the prow job itself does support the plugin which is why we can use it, it just needs to happen manually.